tests/integration/helper.go

```diff
@@ -52,6 +51,7 @@ func Pack() occam.Pack {
 func PackBuild(name string, source string, opts ...BuildOpts) (occam.Image, fmt.Stringer, error) {
 	b := Pack().Build.
 		WithBuilder(*Builder).
+		WithTrustBuilder().
```

Comment:
I found this was necessary or else pack tries to look for a lifecycle image locally and fails. I remember needing to do this for local builders in the past. I believe we can alternately set the pull policy to `if-not-present` (instead of the default `always`) for the `pack build` command, but this is faster and seems appropriate since we are in control of the builder being used.

Reply:
I think line 55 is setting it to pull policy never

Reply:
Who knew the answer was so close! I had a feeling something was doing that somewhere.